.github/workflows/continuous-delivery.yml

```
@@ -42,3 +42,5 @@ jobs:
           tags: |
             ${{ env.REGISTRY }}/${{ github.repository_owner }}/${{ env.SERVICE }}:${{ env.TAG }}
             ${{ env.REGISTRY }}/${{ github.repository_owner }}/${{ env.SERVICE }}:latest
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)